apps/status-api/src/modules/ControllerModule.ts

```diff
@@ -3,6 +3,8 @@ import { ActuatorController } from '@defichain-apps/libs/actuator'
 import { BlockchainStatusController } from '../controllers/BlockchainStatusController'
 import { WhaleApiClient } from '@defichain/whale-api-client'
 import { ConfigService } from '@nestjs/config'
+import { SemaphoreCache } from '../../../whale/src/module.api/cache/semaphore.cache'
```

Comment:
This should probably be extracted out of `whale` as a `lib` module.

Reply:
https://github.com/DeFiCh/jellyfish/issues/1358 created